hpat/tests/test_series.py

```diff
@@ -34,6 +37,31 @@
     ),
 ]]
 
+test_global_input_data_float64 = [
+    [1.0, np.nan, -1.0, 0.0, 5e-324],
+    [np.nan, np.inf, np.NINF, np.NZERO]
+]
+
+min_int64 = -9223372036854775808
+max_int64 = 9223372036854775807
```

Comment:
sys.intmax

Reply:
@densmirn Use numba.targets.builtins.get_type_max_value for this

Reply:
Added.